src/components/Header.js

```diff
@@ -1,13 +1,9 @@
-import Link from 'next/link';
+import {StyledHeader, Headline} from './styled/Header.styled';
 
 export default function Header() {
 	return (
-		<header>
-			<nav>
-				<Link href="/">Home</Link>
-				<Link href="/about">About me</Link>
-				<Link href="/bar">CLICK HERE !! my Bar !! CLICK HERE</Link>
-			</nav>
-		</header>
+		<StyledHeader>
+			<Headline>My Bar</Headline>
+		</StyledHeader>
```

Comment:
For the future I would recommend you to put each styled component in a separate file and not to have multiple styled-components in one file. As it is possible to reuse styled components it could be hard to find the Headline components in you `Header.js` if you may reuse it as part of a component that is not part of the `Header` component. :)